examples/can_print/BUILD

```diff
@@ -0,0 +1,61 @@
+load("//bazel:defs.bzl", "cc_firmware")
+load("//bazel/tools:defs.bzl", "can_api_files", "dbc_gen")
+
+exports_files([
+    "can_print_example.yml",
+])
+
+# Used for debugging with compiler preprocessor output.
+cc_binary(
```

Comment:
still needed?

Reply:
Yes. Because of the heavy use of C macros viewing the `-e` output is invaluable. I'll keep it since it's a rule that's out of the way.